lib/sync.js

```diff
@@ -20,7 +20,7 @@ module.exports = function (x, options) {
     };
     var readFileSync = opts.readFileSync || fs.readFileSync;
 
-    var extensions = opts.extensions || ['.js'];
+    var extensions = opts.extensions || Object.keys (require.extensions);
```

Comment:
the space after `Object.keys` would need to be removed.

Reply:
@ljharb ah i see. Was just going off the current JS Standard of space after function names. https://standardjs.com Updated in https://github.com/browserify/resolve/pull/138/commits/b3270e62231ebf6ad65845a514d37455ade94476

Reply:
While that is named "standard", it is in no way a standard, and in some ways directly contradicts what the vast majority of JS devs do.